File: core/src/test/java/com/ustadmobile/test/core/impl/PlatformTestUtil.java
package com.ustadmobile.test.core.impl;


import com.ustadmobile.port.javase.impl.UmContextSe;

/**
 * TestUtil is designed to abstract away the differences between conducting testing on "smart"
 * devices where we can run NanoHTTPD and J2ME where we need to use an external server.
 */

public class PlatformTestUtil {

    static UmContextSe testContext = new UmContextSe();

    public static Object getTargetContext() {
        return testContext;
    }

    public static Object getTestContext() {
        return new Object();
    }

}

File: core/src/jvmTest/kotlin/com/ustadmobile/test/core/impl/PlatformTestUtil.kt
//package com.ustadmobile.test.core.impl
//
//
//import com.ustadmobile.port.javase.impl.UmContextSe
//
///**
// * TestUtil is designed to abstract away the differences between conducting testing on "smart"
// * devices where we can run NanoHTTPD and J2ME where we need to use an external server.
// */
//
//object PlatformTestUtil {
//
//    internal var testContext = UmContextSe()
//
//    @JvmStatic
//    val targetContext: Any
//        get() = testContext
//
//    @JvmStatic
//    fun getTestContext(): Any {
//        return Any()
//    }
//
//}
